js/index.js

```diff
@@ -40,3 +40,119 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+const imgID =
+document.getElementById("logo-img")
+// console.log(imgID)
+
+const Idimg =
+document.getElementById("cta-img")
+// console.log(Idimg)
+
+const navStyles = 
+document.querySelectorAll("nav a")
+
+navStyles.forEach(element => {
+  element.stlye.color = 'green'
+});
+
+const newCont = 
+document.querySelector('header')
+
+// nav
+
+const NewNav =
+document.createElement("a");
+
+NewNav.textContent =   {
```

Comment:
With text content, you should add a string to the content of the new element. Not an object.